pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -254,16 +262,11 @@ func (c *ClusterMemberController) ensureEtcdLearnerPromotion(ctx context.Context
 		return nil
 	}
 
-	members, err := c.etcdClient.MemberList(ctx)
-	if err != nil {
-		return fmt.Errorf("could not get etcd member list: %v", err)
-	}
-
```

Comment:
this call will be removed with my changes .. so instead of issuing a request to etcd-server, the controller retrieve them locally .. `etcdmemberlistercontroller` makes sure other controllers do not have stale cached data